examples/manipulating_visualizing/plot_prob_atlas.py

```diff
@@ -0,0 +1,47 @@
+"""
+Plotting 4D Atlas maps or regions onto the anatomical image
+===========================================================
+
+We show here how to plot the atlas maps or statistical maps which are
+4D using a three different display types, for example using
+
+1. "contours", ROIs contours are delineated by colored lines.
```

Comment:
I loooked the preview of the doc. It looks so clumbsy.  Thats the reason why I added some spaces. If you are talking about the line gaps. for example between 1. and 2.